### PR TITLE
Replace technet and fwlink

### PR DESCRIPTION
**Pull request by Youssef1313**
Contributes to #2037

### CODE REVIEW

**Comment by IEvangelist** [comment]
What do you think @CamSoper?

File: docs/azure/migration/app-service.md [hunk 1]
```diff
@@ -47,55 +47,71 @@ Azure App Service supports anonymous authentication by default and Forms authent
...
 
 ## Azure App Service with Windows Containers
+
 If your app cannot be migrated directly to App Service, consider App Service using Windows Containers, which enables usage of the GAC, COM components, MSIs, full access to .NET FX APIs, DirectX, and more.
 
 ## See also
 
 * [How to determine if your app qualifies for App Service](https://appmigration.microsoft.com/)
-* [Moving your database to the cloud](https://go.microsoft.com/fwlink/?linkid=863217)
+* [Moving your database to the cloud](https://docs.microsoft.com/dotnet/azure/migration/sql)
```
Comment (gewarren) [inline docs/azure/migration/app-service.md:116]:
```suggestion
* [Moving your database to the cloud](sql.md)
```

File: includes/migration-guide/runtime/data/sqlconnection-can-no-longer-connect-sql-server-1997-databases-using-via.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 |   |   |
 |---|---|
-|Details|Connections to SQL Server databases using the [Virtual Interface Adapter (VIA) protocol](https://technet.microsoft.com/library/ms191229%28v=sql.105%29.aspx) are no longer supported. The protocol used to connect to a SQL Server database is visible in the connection string. A VIA connection will contain via:&lt;servername&gt;. If this app is connecting to SQL via a protocol other than VIA (tcp: or np: for example), then no breaking change will be encountered.Also, connections to SQL Server 7 (1997) are no longer supported.|
+|Details|Connections to SQL Server databases using the [Virtual Interface Adapter (VIA) protocol](https://docs.microsoft.com/previous-versions/sql/sql-server-2008-r2/ms191229(v=sql.105)) are no longer supported. The protocol used to connect to a SQL Server database is visible in the connection string. A VIA connection will contain via:&lt;servername&gt;. If this app is connecting to SQL via a protocol other than VIA (tcp: or np: for example), then no breaking change will be encountered.Also, connections to SQL Server 7 (1997) are no longer supported.|
```
Comment (Thraka) [inline includes/migration-guide/runtime/data/sqlconnection-can-no-longer-connect-sql-server-1997-databases-using-via.md:5]: Spotted a tiny error in the original text.

```suggestion
|Details|Connections to SQL Server databases using the [Virtual Interface Adapter (VIA) protocol](https://docs.microsoft.com/previous-versions/sql/sql-server-2008-r2/ms191229(v=sql.105)) are no longer supported. The protocol used to connect to a SQL Server database is visible in the connection string. A VIA connection will contain via:&lt;servername&gt;. If this app is connecting to SQL via a protocol other than VIA (tcp: or np: for example), then no breaking change will be encountered. Also, connections to SQL Server 7 (1997) are no longer supported.|
```

**Review by mairaw — APPROVED** [review]
Changes look good with the suggestions applied. Thanks @Youssef1313!

**Pull request merged**